acisfp_check.py

```diff
@@ -830,22 +838,31 @@ def emit(self, record):
     rootlogger = logging.getLogger()
     rootlogger.addHandler(NullHandler())
 
+    logger = logging.getLogger('acisfp_check')
+    logger.setLevel(logging.DEBUG)
```

Comment:
Why is this being hard-coded to `DEBUG` here? I noticed that it changed from the previous behavior.